source/hardware/controllers/behringer_bcr2000.rst

```diff
@@ -0,0 +1,106 @@
+Behringer B-Control BCR2000
+===========================
+
+.. figure:: https://mediadl.musictribe.com/media/PLM/data/images/products/P0245/2000Wx2000H/BCR2000_P0245_Top_XL.png
+  :width: 600
+
+- `Manufacturer's product page <https://www.behringer.com/behringer/product?modelCode=P0245>`_
+- `Forum thread <https://mixxx.discourse.group/t/behringer-b-control-bcr2000/20287>`_
+
+The B-CONTROL BCR2000 is a general-purpose USB/MIDI controller containing 20 Buttons, 24 rotary
+encoders and 8 push encoders. Every control is backed by LEDs that show the current value. As a
+pure MIDI controller it contains no interfaces for audio or microphones.
+
+The controller can be be configured extensively.
+It offers 32 presets that may be programmed by the user.
+
+.. versionadded:: 2.3
+
+Compatibility
+-------------
+
+This controller is a class compliant USB MIDI and audio device, so it can be used without any
+special drivers on GNU/Linux, Mac OS X, and Windows.
+
+Setup
+-----
+Controller Preset
+^^^^^^^^^^^^^^^^^
+The default mapping works out-of-the-box for the factory settings of controller preset #1
+(i.e. Channel 1, MIDI control values ``0x01`` to ``0x6C``).
+
+If you don't want to use this preset, you can choose one of the
+following options:
+
+#. Load the dump file ``bcr_Only_Controllers.syx`` via Sysex to any other preset slot.
+   The dump file and instructions how to install it in the controller is available on the
+   manufacturer's webpage.
+#. Factory reset the controller.
+#. Customize the preset mapping (see section *Mapping Description* for details).
+
+Button Behaviour
+^^^^^^^^^^^^^^^^
+It is recommended, but not required, to change the button behaviour from *Toggle On* (default)
+to *Toggle Off*. Both behaviours send an ``On`` message on button press, but the action to
+send an ``Off`` message is different: *Toggle Off* responds to a button release while *Toggle On*
+ignores the release and requires a second press. To change the behaviour:
+
+#. Hold *Edit*, press a button, then release both
+#. Turn the 6th push encoder (labelled *MODE*) until the display shows ``toFF``
+#. Press *Exit*
+
+To make the change persistent, press *Store* twice. See the manual for details.
+
+.. _mapping-description:
+
```

Comment:
This anchor isn't referenced anywhere. It would also conflicts with other mappings because it lacks a `behringer-bcr2000-` prefix.

```suggestion
```

Reply:
Good spot! I used it first but then removed the reference and forgot this one.